armi/reactor/converters/uniformMesh.py

```diff
@@ -792,12 +734,18 @@ class NeutronicsUniformMeshConverter(UniformMeshGeometryConverter):
     back to the source reactor.
```

Comment:
I think this note might be out of date? Or maybe just flat-out wrong? Mapping the parameters from and then back to a source reactor is exactly what we do _not_ want to do. 

Does this need to be updated?

Reply:
Yes it was just the old doc string, I had not updated it